recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb

```diff
@@ -0,0 +1,14 @@
+SUMMARY = "colorzero is a color manipulation library for Python"
```

Comment:
Can we move this to meta-python?

Reply:
By meta-python you mean inside openembedded.org/meta-openembedded/meta-python ?

Reply:
Yes. That is correct. http://layers.openembedded.org/layerindex/branch/master/layer/meta-python/

Reply:
@agherzan  I have sent a patch to meta-python, let's wait till it is merged. Than I will push a new commit here.

Reply:
Thanks!

Reply:
@agherzan I have made the change, the python3-colorzero recipe is merged. 
http://cgit.openembedded.org/meta-openembedded/tree/meta-python/recipes-devtools/python/python3-colorzero_2.0.bb